test/asyncio_tests/__init__.py

```diff
@@ -95,23 +95,31 @@ def setUp(self):
         self.collection = self.db.test_collection
         self.loop.run_until_complete(self.collection.drop())
 
-    def get_client_kwargs(self, **kwargs):
+    def _get_ssl_kwargs(self, **kwargs):
         if env.mongod_started_with_ssl:
             kwargs.setdefault("tlsCAFile", CA_PEM)
             kwargs.setdefault("tlsCertificateKeyFile", CLIENT_PEM)
-
         kwargs.setdefault("tls", env.mongod_started_with_ssl)
-        kwargs.setdefault("io_loop", self.loop)
+        return kwargs
 
+    def _get_io_loop_kwargs(self, **kwargs):
+        kwargs.setdefault("io_loop", self.loop)
         return kwargs
 
-    def asyncio_client(self, uri=None, *args, **kwargs):
+    def get_client_kwargs(self, set_loop=True, **kwargs):
+        if set_loop:
+            kwargs = self._get_io_loop_kwargs(**kwargs)
+        return self._get_ssl_kwargs(**kwargs)
```

Comment:
Now that we have set_loop, can we go back to using a single get_client_kwargs method?